simplq/src/components/pages/Status/QueueStatus.jsx

```diff
@@ -50,7 +50,7 @@ function QueueStatus(props) {
         handleApiErrors(err);
         timeoutId = setTimeout(update, TIMEOUT);
       });
-  }, [tokenId, oldTokenStatus]);
+  }, [tokenId, oldTokenStatus, showNotification]);
 
```

Comment:
Added this to prevent a warning. Not 100% sure if it's required or not.

Reply:
I've commented, we should remove this for now. Fixing this will make the code ugly.